src/text-editor.coffee

```diff
@@ -2977,6 +2972,14 @@ class TextEditor extends Model
 
   logScreenLines: (start, end) -> @displayBuffer.logLines(start, end)
 
+  emitWillInsertTextEvent: (text) ->
+    result = true
+    cancel = -> result = false
+    willInsertEvent = {cancel, text}
+    @emit('will-insert-text', willInsertEvent) if includeDeprecatedAPIs
```

Comment:
Do we still need `includedDeprecatedAPIs` post 1.0? Is it just because we haven't removed core's dependency on `emissary` yet?